### DISCUSSION TITLE
Typical bugs in previous versions of Security Onion

### DISCUSSION THREAD

**Discussion by OldDriver0907** [Security-Onion-Solutions/securityonion#2459] [Unsupported Versions] [answered]
I'm a college student and I'm doing my FYP about typical bugs in previous versions of Security Onion, better related to network security. 
I'd like to know some about it. If it can be answered, please tell me some in a particular version.

**Comment by dougburks** [comment, marked as answer]
If I were your professor, I would want you to research that yourself.

**Reply by OldDriver0907** [reply]
Thanks and I got some from CVE.
